Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/MainActivity.java

```diff
@@ -162,7 +164,9 @@ protected void onPostExecute(String s) {
         parseServer();
       } catch (Exception e) {
         Log.d(tag, "Parsing server cache failed");
-        Log.d(tag, String.valueOf(e.getCause()));
+        Log.d("Rokus Logs:", String.valueOf(e.getCause()));
+        Log.d("Rokus Logs:", e.getMessage());
```

Comment:
Can't you just Log.d("SLATE", "Parsing server message failed", e); ?

Why do you print the exception fields explicitly?